easybuild/tools/systemtools.py

```diff
@@ -338,6 +338,9 @@ def get_os_version():
                     ('2.6.32', '_SP1'),
                     ('3.0', '_SP2'),
                 ],
+                "12": [
+                    ('3.12', ''),
```

Comment:
good enough for now, but maybe this will have to be made stricter once SLES 12 SP1 is out...

maybe add a comment on that?